docs/guides/flux-v1-migration.md

```diff
@@ -6,7 +6,16 @@ Read the [FAQ](../faq/index.md) to find out what differences are between v1 and
 !!! info "Automated image updates"
     The image automation feature is under development in Flux v2.
     Please consult the [roadmap](../roadmap/index.md) for more details.
-    
+
+!!! info "Feature parity"
+    "Feature parity" does not mean Flux v2 works exactly the same as v1 (or is
+    backward-compatible); it means you can accomplish the same results, while
+    accounting for the fact that it's a system with a substantially different
+    design.
+    This may at times mean that you have to make adjustments to the way your
+    current cluster configuration is structured. If you feel this is you,
+    and you need help, please refer to the [support page](https://fluxcd.io/support/).
```

Comment:
```suggestion
    current cluster configuration is structured. If you are in this situation and
    need help, please refer to the [support page](https://fluxcd.io/support/).
```

"this" referring to the implicit situation of having to adjust configuration is a bit unclear; the above makes it explicit. Could also cut out some verbiage with

```suggestion
    current cluster configuration is structured. If you need help with this, please refer to the [support page](https://fluxcd.io/support/).
```

Reply:
To me the verbiage sounded more friendly and/or sympathetic, so I opted for suggestion 1. Thank you :100: